### ISSUE TITLE
CCS Style Sheets added to doc not working correctly.

### ISSUE THREAD

**Issue by HippyCraig** [NicoNekoru/obsidan-advanced-table-xt#31]
I have a style sheet setup in obsidian snippets
```
.ccg-work-table-top-rounded-install-doc {
        ......... Lots of other styles removed for clarity
	& .instruction-separator {
		background-color: rgb(110, 110, 110) !important;
	}
}
```

Inside the table in the MD view I have this:
```
| Patch installs ~ .instruction-separator | <                               |
```

Looking at the rendered HTML in Chrome it appears to keep the . in the name of the class when it shouldnt be there.
```
<td class=".instruction-separator" row-index="3" col-index="0" colspan="2" style="text-align: left;"><p> Patch installs </p></td>
```
This doesnt work but if I mod the value in chrome directly like:
```
<td class="instruction-separator" row-index="3" col-index="0" colspan="2" style="text-align: left;"><p> Patch installs </p></td>
```
The class name is evaluated correctly and creates the desired effect.   Is there something I need to change in my MD to make this work correctly?

**Comment by NicoNekoru** [comment]
Ah yeah this was intended when I initially made the function since the "classes" were not actually creating a stylesheet, but instead mapping the styles from an object. I didn't realize that this functionality could be useful and the fix should be pretty simple.

**Comment by NicoNekoru** [comment]
Fix landed in 1.2.4
